panel/_templates/index.html

```diff
@@ -1,276 +1,200 @@
-	vertical-align: middle;
-	float: right;
-	height: 20px;
-	color: white;
-	margin-top: 10px;
-	margin-right: 10px;
-	fill: var(--neutral-foreground-rest);
-    }
-    .card-image {
-	height: 175px;
-	width: 100%;
-	margin-top: 25px;
-    }
-    object {
-        height: 175px;
-	max-width: calc(100% - 50px);
-	margin-top: 25px;
-        border-radius: calc(var(--control-corner-radius) * 1px);
-    }
-    .card-content {
-	padding: 10px 10px 10px;
-	color: var(--neutral-foreground-rest);
-    }
-    .card-text {
-	height: 100px;
-    }
-    .card-header {
-	height: 2em;
-	text-align: center;
-    }
-    footer {
-	padding: .50rem;
-	text-align: center;
-	font-size: .75rem;
-    }
-    #panel-logo {
-	width: 300px;
-    }
-    .card-link {
-	text-decoration: none;
-	color: var(--neutral-foreground-rest);
-    }
-  </style>
-</head>
-<body>
-  <fast-design-system-provider id="body-design-provider" use-defaults>
-    <fast-design-system-provider id="header-design-provider" use-defaults>
-      <section class="header">
-        <fast-switch id="theme-switch" style="float: right; padding-top: 1em; padding-right: 2em;" onChange="toggleLightDarkTheme()" checked>
-          <span slot="checked-message">
-            <svg class="theme-toggle-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25"><path d="M0 0h24v24H0z" fill="none"/><path d="M6.76 4.84l-1.8-1.79-1.41 1.41 1.79 1.79 1.42-1.41zM4 10.5H1v2h3v-2zm9-9.95h-2V3.5h2V.55zm7.45 3.91l-1.41-1.41-1.79 1.79 1.41 1.41 1.79-1.79zm-3.21 13.7l1.79 1.8 1.41-1.41-1.8-1.79-1.4 1.4zM20 10.5v2h3v-2h-3zm-8-5c-3.31 0-6 2.69-6 6s2.69 6 6 6 6-2.69 6-6-2.69-6-6-6zm-1 16.95h2V19.5h-2v2.95zm-7.45-3.91l1.41 1.41 1.79-1.8-1.41-1.41-1.79 1.8z"/></svg>
-          </span>
-          <span slot="unchecked-message">
-            <svg class="theme-toggle-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25"><path d="M0 0h24v24H0z" fill="none"/><path d="M10 2c-1.82 0-3.53.5-5 1.35C7.99 5.08 10 8.3 10 12s-2.01 6.92-5 8.65C6.47 21.5 8.18 22 10 22c5.52 0 10-4.48 10-10S15.52 2 10 2z"/></svg>
-          </span>
-        </fast-switch>
-        <fast-tooltip anchor="theme-switch">Click to toggle the Theme</fast-tooltip>
-        <div class="header-content">
-          <div>
-            <fast-anchor id="title" href="https://panel.holoviz.org" appearance="neutral" target="_self">
-              <img id="panel-logo" src="https://panel.holoviz.org/_static/logo_horizontal.png"/>
-            </fast-anchor>
-            <fast-tooltip anchor="title">Click to visit the Panel web site</fast-tooltip>
-          </div>
-          <span id="subtitle">Running Applications</span>
+      }
+      setSearchFromParams()
+    </script>
+  </head>
+  <body>
+    <section class="header">
+      <div class="header-content">
+        <div>
+          <img id="panel-logo" src="{{ PANEL_CDN }}images/logo_horizontal.png"/>
+        </div>
+        <div class="title-area">
+          <span id="title">Running Applications</span>
```

Comment:
Is this needed? Maybe can remove since serving num apps is the same?

Reply:
Agreed, will find alternative wording.